source/pdu/main.c

```diff
@@ -187,6 +199,8 @@ int main()
     taskCreate(daqPeriodic, DAQ_UPDATE_PERIOD);
     taskCreateBackground(canTxUpdate);
     taskCreateBackground(canRxUpdate);
+    taskCreate(updateCurrent, 15);
+    taskCreate(updateVoltage, 15);
```

Comment:
can you combine these into 'auto_switch_periodic' so that there is just one task